x/perpetual/keeper/query_open_estimation.go

```diff
@@ -105,7 +106,7 @@ func (k Keeper) HandleOpenEstimation(ctx sdk.Context, req *types.QueryOpenEstima
 		//getting custody
 		if mtp.CollateralAsset == baseCurrency {
 			leveragedAmtTokenIn := sdk.NewCoin(mtp.CollateralAsset, leveragedAmount)
-			custodyAmount, slippage, weightBreakingFee, err = k.EstimateSwapGivenIn(ctx, leveragedAmtTokenIn, mtp.CustodyAsset, ammPool)
+			custodyAmount, slippage, weightBreakingFee, err = k.EstimateSwapGivenIn(ctx, leveragedAmtTokenIn, mtp.CustodyAsset, ammPool, mtp.Address)
```

Comment:
The mtp address is empty here as it's just a query. Either we get address or we send default. If we get address from query, then in response need to send final fees

Reply:
yes, I have handled it later. we use default value from tier module

Reply:
quick question, should this be here `mtp.Address` or user address?

Reply:
@amityadav0 @Wgil user address

Reply:
mtp.address is user address when opening a position, mtp.address is set to msg.creator